### PR TITLE
Docs: Perlmutter

### PR DESCRIPTION
**Pull request by ax3l**
Start a documentation page for Perlmutter.

### CODE REVIEW

**Review by kngott — APPROVED** [review]

File: Docs/source/install/hpc/perlmutter.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+Perlmutter (NERSC)
+==================
+
+.. warning::
+
+   Perlmutter is still in acceptance testing.
+   This page documents our internal testing workflow only.
+
+The `Perlmutter cluster <https://www.nersc.gov/systems/perlmutter/>`_ is located at NERSC.
```
Comment (ax3l) [inline Docs/source/install/hpc/perlmutter.rst]: Better page:
```suggestion
The `Perlmutter cluster <https://docs.nersc.gov/systems/perlmutter/>`_ is located at NERSC.
```

File: Docs/source/install/hpc/perlmutter.rst [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+If you are new to this system, please see the following resources:
+
+* `NERSC user guide <https://docs.nersc.gov/>`__
+* Batch system: `Slurm <https://docs.nersc.gov/jobs/>`__
+* `Jupyter service <https://docs.nersc.gov/services/jupyter/>`__
+* `Production directories <https://www.nersc.gov/users/storage-and-file-systems/>`__:
```
Comment (ax3l) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
* `Production directories <https://docs.nersc.gov/filesystems/perlmutter-scratch/>`__:
```

File: Docs/source/install/hpc/perlmutter.rst [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+   Perlmutter is still in acceptance testing.
+   This page documents our internal testing workflow only.
+
+The `Perlmutter cluster <https://www.nersc.gov/systems/perlmutter/>`_ is located at NERSC.
+
+If you are new to this system, please see the following resources:
+
+* `NERSC user guide <https://docs.nersc.gov/>`__
+* Batch system: `Slurm <https://docs.nersc.gov/jobs/>`__
```
Comment (ax3l) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
* Batch system: `Slurm <https://docs.nersc.gov/systems/perlmutter/#running-jobs>`__
```

File: Docs/source/install/hpc/perlmutter.rst [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+* `Jupyter service <https://docs.nersc.gov/services/jupyter/>`__
+* `Production directories <https://www.nersc.gov/users/storage-and-file-systems/>`__:
+
+  * ``$SCRATCH``: per-user production directory (20TB)
```
Comment (ax3l) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
  * ``$PSCRATCH``: per-user production directory (<TBD>TB)
```

**Review by ax3l — COMMENTED** [review]
More specific doc pages

File: Docs/source/install/hpc/perlmutter.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+   # module load nano
+
+   # optional: FFT, I/O, Python, ...
+   # TODO
+
+   # optional: an alias to request an interactive node for two hours
+   function getNode() {
+       salloc -N 1 --ntasks-per-node=4 -t 2:00:00 -C gpu -c 32 -G 4 -A $proj
+   }
+
```
Comment (ax3l) [inline Docs/source/install/hpc/perlmutter.rst:66]:
```suggestion

   # GPU-aware MPI
   export MPICH_GPU_SUPPORT_ENABLED=1
```

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**